llarp/quic/tunnel.cpp

```diff
@@ -538,7 +538,7 @@ namespace llarp::quic
           [this,
            after_path = std::move(after_path),
            pport = pport,
-           remote_addr = std::move(remote_addr)](auto maybe_remote) {
+           remote_addr = remote_addr](auto maybe_remote) {
```

Comment:
No need for the `x = x` here, just `x` does the same.

Reply:
yeah, fair point.  Was a lazy/quick fix, will amend.